src/main/java/com/epam/javacourses2016/task9/SolverTask9.java

```diff
@@ -17,6 +21,29 @@
      * @return Множество полученных слов.
      */
     public HashSet<String> getUniqueWords(File input) {
-        return null;
+        HashSet<String> result = new HashSet<>();
+        ArrayList<String> allWords = new ArrayList<>();
+        try {
+            FileReader fileInputStream = new FileReader(input);
+            BufferedReader reader = new BufferedReader(fileInputStream);
+            String word;
+            while ((word = reader.readLine()) != null) {
+                allWords.add(word);
```

Comment:
We don't need that collection, one single hash set with custom Comparator should be enough